integration/nwo/deploy.go

```diff
@@ -499,7 +499,29 @@ func EnableCapabilities(network *Network, channel, capabilitiesGroup, capabiliti
 		),
 	}
 
-	UpdateConfig(network, orderer, channel, config, updatedConfig, false, peers[0], peers...)
+	UpdateConfig(network, orderer, channel, config, updatedConfig, false, peers[0], nil, peers...)
+}
+
+func EnableChannelCapabilities(network *Network, channel, capabilitiesVersion string, getConfigBlockFromOrderer bool, orderer *Orderer, ordererSigners []*Orderer, peerSigners ...*Peer) {
+	if len(peerSigners) == 0 {
+		return
+	}
+
+	config := GetConfig(network, peerSigners[0], orderer, channel)
+	updatedConfig := proto.Clone(config).(*common.Config)
+
+	updatedConfig.ChannelGroup.Values["Capabilities"] = &common.ConfigValue{
+		ModPolicy: "Admins",
+		Value: protoutil.MarshalOrPanic(
+			&common.Capabilities{
+				Capabilities: map[string]*common.Capability{
+					capabilitiesVersion: {},
```

Comment:
I'm confused, where do you enable the V3_0 capability here?

Reply:
I followed the example of [./integration/e2e/e2e_test.go#L197-L198](https://github.com/hyperledger/fabric/blob/05b604cc64eeed2684e917a7d34730d876a2ee59/integration/e2e/e2e_test.go#L197-L198), which is a pattern repeated in many integration tests.  V3_0 capabilities are enabled by line **518**.

Reply:
oh I see, the capabilitiesVersion you pass is set to V3_0